Code/GraphMol/ChemReactions/catch_tests.cpp

```diff
@@ -1101,7 +1101,19 @@ TEST_CASE("CXSMILES for reactions", "[cxsmiles]") {
     CHECK(rxn->getReactants()[1]->getAtomWithIdx(2)->getPropIfPresent(
         common_properties::atomLabel, alabel));
     CHECK(alabel == "_AP1");
+
+    auto roundtrip = v2::ReactionParser::ReactionFromSmarts(ChemicalReactionToRxnCXSmarts(*rxn));
```

Comment:
Doing the round trip is a great test, but I think it would be nice to also have a check or two to verify that the output string actually contains the CX extensions in the correct format.

This same comment applies to the other tests as well.